### PR TITLE
Implement rule parser

### PR DESCRIPTION
**Pull request by piotrkaminski95**
check it please

### CODE REVIEW

File: src/main/java/com/codecool/expertsystem/models/RuleParser.java [hunk 1]
```diff
@@ -17,6 +17,8 @@ public RuleParser(String xmlPath) {
     public RuleRepository getRuleRepository() {
         RuleRepository ruleRepository = new RuleRepository();
         NodeList nodeList = doc.getElementsByTagName("Rule");
```
Comment (Senseinvader) [inline src/main/java/com/codecool/expertsystem/models/RuleParser.java]: Have you initialized doc somewhere else?

File: src/main/java/com/codecool/expertsystem/models/RuleParser.java [hunk 2]
```diff
@@ -17,6 +17,8 @@ public RuleParser(String xmlPath) {
...
+        for (int i = 0; i < nodeList.getLength(); i++) {
```
Comment (Senseinvader) [inline src/main/java/com/codecool/expertsystem/models/RuleParser.java]: extra empty loop

File: src/main/java/com/codecool/expertsystem/models/RuleParser.java [hunk 3]
```diff
@@ -1,13 +1,62 @@
...
 
+    public RuleParser(String xmlPath) {
+        loadXMLDocument(xmlPath);
+    }
 
     public RuleRepository getRuleRepository() {
-        return null;
+        RuleRepository ruleRepository = new RuleRepository();
+        NodeList nodeList = doc.getElementsByTagName("Rule");
+        for (int i = 0; i < nodeList.getLength(); i++) {
```
Comment (WojTechM) [inline src/main/java/com/codecool/expertsystem/models/RuleParser.java]: Empty for loop - implement or delete

File: src/main/java/com/codecool/expertsystem/models/RuleParser.java (hunk 1 above)
Comment (szymslow12) [reply src/main/java/com/codecool/expertsystem/models/RuleParser.java]: doc is initialized in XMLParser in loadXMLDocument method

**Pull request merged**